netket/graph/hypergrid.py

```diff
@@ -0,0 +1,146 @@
+from graph import Graph
+
+import numpy as _np
+import networkx as _nx
+
+
+class Hypergrid(Graph):
```

Comment:
This PR now seems to include `HyperGrid` and `Hypergrid` classes in `grid.py` and `hypergrid.py`, respectively. That's probably a leftover from previous changes, right?

Reply:
In my `hypergrid.py` file I just have `Hypergrid`, `Hypercube`, `Square` and `Chain`. I don't know why `HyperGrid` is appearing.

Reply:
maybe you didn't commit it? try git status to see what's going on!

Reply:
> On branch v3.0
> Your branch is up-to-date with 'origin/v3.0'.
> 
> nothing to commit, working tree clean

It looks good